Tests/Selenium.Automation.UI/Filter/FilterPage.cs

```diff
@@ -34,6 +34,8 @@ public FilterCheckBox[] GetFilterCheckBoxes(string name)
             this.FindAll<HtmlLink>(new Locator(How.XPath, ".//rz-filter-stack//div[contains (@class, 'sidebar-block__toggle-title')]"))
```

Comment:
Page means Page object not any methods should placed in PO for the methos you have Steps
regular view of PO is 
`	public class MainPage : WebPage
	{
		public MainPage(IWebDriver webDriver)
			: base(webDriver)
		{
		}

		[FindBy(How.XPath, ".//rz-user//button")]
		public HtmlButton OpenLoginButton { get; set; }

		[FindBy(How.XPath, ".//header")]
		public HeaderSection HeaderSection { get; set; }

		[FindBy(How.XPath, ".//rz-product-top")]
		public ProductTopElement ProductTopElement { get; set; }

		[FindBy(How.XPath, ".//rz-product-top")]
		public DocSection DocSection { get; set; }
	}`
	
	but not 
	`    public class FilterPage : WebPage
    {
        public FilterPage(IWebDriver webDriver) : base(webDriver)
        {
        }

        private HtmlElement GetFilterByCat(string category)
        {
            string locator = $".//div[@data-filter-name='{category}']";
            HtmlElement element = Find<HtmlElement>(new Locator(How.XPath, locator));
            return element;
        }

        public FilterCheckBox[] GetFilterCheckBoxes(string name)
        {
            var filterCheckBoxes = GetFilterByCat(name).FindAll<FilterCheckBox>(new Locator(How.XPath, ".//li"));

            return filterCheckBoxes.ToArray();
        }

        public HtmlLink[] FilterCategories =>
            this.FindAll<HtmlLink>(new Locator(How.XPath, ".//rz-filter-stack//div[contains (@class, 'sidebar-block__toggle-title')]"))
            .ToArray();

        public object Url => "ua/consoles/c80020/";

        public string[] FilterCategoriesLists()
        {
            List<string> filterCatList = new List<string>();
            foreach (HtmlLink element in FilterCategories)
            {
                filterCatList.Add(element.ToString());

            }
            return filterCatList.ToArray();
        }

        public HtmlElement SelectedFiltersView =>
        Find<HtmlElement>(new Locator(How.XPath, ".//rz-selected-filters"));

        public HtmlLink[] CatalogSelectedItem =>
            this.FindAll<HtmlLink>(new Locator(How.XPath, ".//rz-catalog-settings//li[a]"))
            .ToArray();
    }`

Reply:
you need to move all methods to steps, only elements as property should be on Page model

Reply:
why object type?
`public object Url => "ua/consoles/c80020/";`